spec/dummy/config/initializers/fae.rb

```diff
@@ -12,6 +12,7 @@
     ja: 'Japanese'
   }
 
+  # has_top_nav has been deprecated, but keeping in to ensure definition doesn't rasie exceptions
```

Comment:
All of the `@deprecation`s have been noting when the feature will be deprecated to easily remove it come 2.0. I think the changelog is the best place to track features that have been deprecated.